src/components/InstancesList/InstancesList.js

```diff
@@ -765,29 +773,31 @@ class InstancesList extends React.Component {
       'publishers': r => (r?.publication ?? []).map(p => (p ? `${p.publisher} ${p.dateOfPublication ? `(${p.dateOfPublication})` : ''}` : '')).join(', '),
       'publication date': r => r.publication.map(p => p.dateOfPublication).join(', '),
       'contributors': r => formatters.contributorsFormatter(r, data.contributorTypes),
+      'subject': r => getFullMatchRecord(r?.subject, r.isAnchor),
       'callNumber': r => getFullMatchRecord(r?.fullCallNumber, r.isAnchor),
       'numberOfTitles': r => getFullMatchRecord(r?.totalRecords, r.isAnchor),
-      'subject': r => getFullMatchRecord(r?.subject, r.isAnchor),
     };
 
     const visibleColumns = this.getVisibleColumns();
     const columnMapping = this.getColumnMapping();
 
     const onChangeIndex = (e) => {
-      const isBrowseOptionSelected = Object.values(browseModeOptions).includes(e.target.value);
-      if (isBrowseOptionSelected) {
-        this.setState({ browseSelected: true });
-      } else {
-        this.setState(
-          { browseSelected: false }
-        );
-      }
+      this.setState({ browseSelected: e.target.value });
     };
 
-    const customPaneSubTextBrowse = browseSelected ? <FormattedMessage id="ui-inventory.title.subTitle.browseCall" /> : null;
-    const searchFieldButtonLabelBrowse = browseSelected ? <FormattedMessage id="ui-inventory.browse" /> : null;
-    const titleBrowse = browseSelected ? <FormattedMessage id="ui-inventory.title.browseCall" /> : null;
-    const notLoadedMessageBrowse = browseSelected ? <FormattedMessage id="ui-inventory.notLoadedMessage.browseCall" /> : null;
+    const browseFilter = () => {
+      const { renderer } = getFilterConfig();
+      if (browseSelected === browseModeOptions.SUBJECTS) {
+        return renderer;
+      } else return renderFilters;
```

Comment:
else isn't required here, just return renderFilters

Reply:
Thanks! Fixed it already